nmap.go

```diff
@@ -56,13 +56,25 @@ func NewScanner(options ...func(*Scanner)) (*Scanner, error) {
 
 // Run runs nmap synchronously and returns the result of the scan.
 func (s *Scanner) Run() (result *Run, warnings []string, err error) {
-	var stdout, stderr bytes.Buffer
+	var (
+		stdout, stderr bytes.Buffer
+		f              bool
```

Comment:
Can you name the `f` boolean differently to reflect its actual purpose?

WDYT of `resume` or smth like that?

Reply:
ok! i'll name it `resume `today